src/components/phase-banner/index.njk

```diff
@@ -51,7 +51,7 @@
       ],
       [
         {
-          text: 'phaseTagText'
+          text: 'TagText'
```

Comment:
The arguments table uses the old macro. We should update this file to use the new macro as follows:
- there is no need to import the macro on line 15 `{% from "table/macro.njk" import govukTable %}` as we already import it on the main component template
- `govukTable()` takes a single object argument

Reply:
This has been fixed in the latest commit.